### PR TITLE
2506 document public preview features

### LINKED ISSUE #2506: Document public preview features
- [ ] Snowflake sink
- [ ] Dynamodb sink
- [ ] Bigquery sink
- [ ] Sink Decoupling (https://docs.risingwave.com/docs/current/data-delivery/#sink-decoupling)

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Related doc issue

Resolves https://github.com/risingwavelabs/risingwave-docs/issues/2506

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: src/pages/product-lifecycle.md [hunk 1]
```diff
@@ -28,8 +28,11 @@ The following is a list of all features in the public preview phase:
 | [Subscription](/docs/next/subscription) | Yes                     | 2024.5 | 1.9         | \        |
 | [Time travel queries](/docs/next/time-travel-queries/)                     | Yes                     | 2024.7 | 2.0         | \        |
 | [Manage secrets](/docs/next/manage-secrets/)                          | Yes                     | 2024.7 | 2.0         | \        |
-| [Ingest from MySQL CDC (Auto-map schema)](/docs/next/ingest-from-mysql-cdc/#automatically-map-upstream-table-schema) | Yes                     | 2024.6 | 1.10         | \        |
-|[Ingest data from SQL Server CDC](/docs/next/ingest-from-sqlserver-cdc/)| Yes | 2024.8 | 2.0 | \ |
+| [MySQL CDC source (Auto-map schema)](/docs/next/ingest-from-mysql-cdc/#automatically-map-upstream-table-schema) | Yes                     | 2024.6 | 1.10         | \        |
+| [SQL Server CDC source](/docs/next/ingest-from-sqlserver-cdc/) | Yes | 2024.8 | 2.0 | \ |
+|[Snowflake sink](/docs/current/sink-to-snowflake/)| Yes | 2024.5 | 1.9 | \ |
```
Comment (ShanlanLi) [inline src/pages/product-lifecycle.md:33]: @lmatz  To double-check. We added the Snowflake sink a very long time ago. Is it still in public preview?

File: src/pages/product-lifecycle.md [hunk 2]
```diff
@@ -28,8 +28,11 @@ The following is a list of all features in the public preview phase:
...
+| [Amazon DynamoDB sink](/docs/current/sink-to-dynamodb/) | Yes | 2024.6 | 1.10 | \ |
+| [Google BigQuery sink](/docs/current/sink-to-bigquery/) | Yes | 2023.11 | 1.4 | \ |
```
Comment (ShanlanLi) [inline src/pages/product-lifecycle.md]: @lmatz  To double-check. We added the BigQuery sink a very long time ago. Is it still in public preview?

**Comment by lmatz** [comment]
yes I confirm

see https://github.com/risingwavelabs/risingwave-docs/issues/2336 and https://github.com/risingwavelabs/risingwave-docs/pull/2413

it will needs to be updated

File: src/pages/product-lifecycle.md (hunk 2 above)
Comment (lmatz) [reply src/pages/product-lifecycle.md]: yes, I confirm.

It is being improved continuously, see https://github.com/risingwavelabs/risingwave/pulls?q=is:pr+is:open+bigquery

**Review by lmatz — APPROVED** [review]

File: src/pages/product-lifecycle.md (hunk 2 above)
Comment (WanYixian) [inline src/pages/product-lifecycle.md]:
```suggestion
|[Snowflake sink](/docs/next/sink-to-snowflake/)| Yes | 2024.5 | 1.9 | \ |
| [Amazon DynamoDB sink](/docs/next/sink-to-dynamodb/) | Yes | 2024.6 | 1.10 | \ |
| [Google BigQuery sink](/docs/next/sink-to-bigquery/) | Yes | 2023.11 | 1.4 | \ |
```

**Review by WanYixian — APPROVED** [review]
We can direct all pages here to the upcoming version, so that users can always keep align with the latest changes.

File: src/pages/product-lifecycle.md (hunk 2 above)
Comment (ShanlanLi) [reply src/pages/product-lifecycle.md]: Will  there be conditions like the feature has different stages between the current and next version?

**Pull request merged**